qemu/tests/numa_cpu.py

```diff
@@ -48,7 +55,7 @@ def numa_cpu_guest():
         """
         error_context.context("Get cpus in guest os", test.log.info)
         numa_cpu_guest = []
-        if vm_arch in ('ppc64', 'ppc64le'):
+        if vm_arch in ('aarch64', 'ppc64', 'ppc64le'):
```

Comment:
IIUC, ARM also use the ACPI Component Architecture, this means we can also use acpidump to dump SRAT/PPTT info. So you can try to find the mapping from `acpidump -n SRAT -b` or `acpidump -n PPTT -b`, and I will contact Gavin about this.

Reply:
OK, I think aarch64 can also use SRAT method, the only difference is the keyword.

Reply:
Thanks a lot, I did some tests and it works.
I have only one question, in fact, the if and else methods here are applicable to arm, but we choose to use the else method, right?

Reply:
Yes, both methods work on ARM, but using SRAT should be more directly, because the info is from firmware.